src/modules/project/pages/projectView/views/projectActivityPanel/screens/info/components/ProjectBalanceDisplay.tsx

```diff
@@ -78,17 +85,21 @@ export function ProjectBalanceDisplay({ defaultGoalId, balance, balanceUsdCent,
           <HStack w="100%" display="flex" justifyContent="start" alignItems="center">
             <H1 fontSize="35px">
               {priorityGoal.currency === ProjectGoalCurrency.Usdcent && (
-                <Text as="span" color="neutral.600" fontWeight={500} fontSize="32px">
-                  {'$'}
-                </Text>
+                <>
+                  <Text as="span" color="neutral.600" fontWeight={500} fontSize="32px">
+                    {'$'}
+                  </Text>
+                  {numberWithCommas(centsToDollars(priorityGoal.amountContributed) ?? 0)}
+                </>
```

Comment:
we already had another function that does this called, commaFormatted(), let's use one and get rid of the other, it might create confusion later on.